src/main/java/Deadline.java

```diff
@@ -1,17 +1,34 @@
 import java.time.LocalDate;
 import java.time.format.DateTimeFormatter;
 
+/**
+ * Represents a Deadline task which has a description, a deadline date and a time.
+ */
 public class Deadline extends Task {
 
     protected LocalDate date;
     protected String time;
 
+    /**
```

Comment:
Good work on adding in JavaDocs! Although it did not follow the standard specified format given, it is short, clear and gets the job done.